plugins/modules/idrac_server_config_profile.py

```diff
@@ -738,8 +794,6 @@ def export_scp_redfish(module, idrac):
 
 def wait_for_response(scp_resp, module, share, idrac):
     task_uri = scp_resp.headers["Location"]
-    job_id = task_uri.split("/")[-1]
```

Comment:
why remove this code? is it being handled somewhere else?

Reply:
It was not used anywhere. Removed it to fix sonar issue